### PR TITLE
refactor: generate image with sharp

### PR DESCRIPTION
**Pull request by fpasquet**

### CODE REVIEW

**Review by ElevenMarianne — APPROVED** [review]

File: src/translations/en.translations.json [hunk 1]
```diff
@@ -103,7 +103,7 @@
...
         },
         "title": "Our articles and feedbacks about PHP programming",
-        "description": "<strong>PHP is a backend-oriented programming language</strong> that allows the development of dynamic and interactive applications. It is a popular language, used by a large number of web applications widely used by the general public. In this category, find all the articles, feedbacks and tutorials from our astronauts about <strong>PHP, Symfony, Laravel, the essential tools to improve your productivity, and our reports on major community events!</strong> Good reading!",
+        "description": "<strong>PHP is a backend-oriented programming language</strong> that allows the development of dynamic and interactive applications. It is a popular language, used by a large number of web applications widely used by the general public. In this category, find all the articles, feedbacks and tutorials from our astronauts about <strong>PHP, Symfony, Laravel, Angular, the essential tools to improve your productivity, and our reports on major community events!</strong> Good reading!",
```
Comment (ch3ric) [inline src/translations/en.translations.json:106]: why is 'angular' mixed up with PHPs stuff here?

File: src/helpers/assetHelper.ts [hunk 2]
```diff
@@ -17,59 +30,69 @@ export const getCoverPath = ({
...
   const directoryPath = dirname(path);
   const filename = basename(path, extname(path));
-  const imageFormat = IMAGE_FORMATS[device][format];
+  const imageFormat = SIZES_BY_IMAGE_FORMAT[device][format];
 
   const pathFile = isProd
-    ? `${directoryPath}/${filename}-w${imageFormat.width * pixelRatio}-h${imageFormat.height * pixelRatio}.avif`
-    : `${path}?width=${imageFormat.width * pixelRatio}&height=${imageFormat.height * pixelRatio}&format=avif`;
+    ? `${directoryPath}/${filename}-w${imageFormat.width}-h${imageFormat.height}-x${pixelRatio}.${extension}`
+    : `${path}?width=${imageFormat.width}&height=${imageFormat.height}&pixelRatio=${pixelRatio}&position=${position}&format=${extension}`;
```
Comment (ch3ric) [inline src/helpers/assetHelper.ts:51]: why is the path different depending on the env? I would keep only one image generation process for all envs, as it is in prod

File: src/translations/fr.translations.json [hunk 3]
```diff
@@ -88,7 +88,7 @@
           "description": "Tous nos articles de blog, REX et tutoriels autour du développement Javascript : React.js, Node.js, Nest.js, Next.js, Vue.js, Svelte.js"
         },
```
Comment (ch3ric) [inline src/translations/fr.translations.json:89]: Translations are downloaded from Loco when the app is built, right?
So maybe we don't need to commit these json files?

File: src/translations/en.translations.json (hunk 1 above)
Comment (fpasquet) [reply src/translations/en.translations.json:106]: La modification devait être fait sur Loco mais elle a été fait que FR apparament

File: src/helpers/assetHelper.ts (hunk 2 above)
Comment (fpasquet) [reply src/helpers/assetHelper.ts:51]: Car en local, on ne souhaite pas perdre de temps à générer les images de plus en local on est full SSR et non en static

**Pull request merged**